components/client/src/test/unit/java/com/hotels/styx/client/StyxHttpClientTest.java

```diff
@@ -448,6 +450,10 @@ public void prefersRestrictedOriginsOverStickyOriginsWhenBothAreConfigured() {
         assertThat(argPreferences.getValue().preferredOrigins(), is(Optional.of("Origin-Y")));
     }
 
+    private HttpHandler toHandler(StyxHostHttpClient hostClient) {
```

Comment:
Do you think we could improve the method name? I feel it's not clear enough but I don't know how we could improve it...

Reply:
I wouldn't bother. It is a private method inside a test class.